wgsl/index.bs

```diff
@@ -4033,82 +4033,158 @@ WGSL [=predeclared|predeclares=] an [=enumerant=] for each of the texel formats
 
 ### Sampled Texture Types ### {#sampled-texture-type}
 
-<pre class='def'>
-`texture_1d<type>`
-`texture_2d<type>`
-`texture_2d_array<type>`
-`texture_3d<type>`
-`texture_cube<type>`
-`texture_cube_array<type>`
-</pre>
-* `type` [=shader-creation error|must=] be `f32`, `i32`, or `u32`
+A <dfn noexport dfn-for='type'>sampled texture</dfn> is capable
+of being [=memory access|accessed=] in conjunction with a [=type/sampler=].
+It can also be accessed without the use of a sampler.
+Sampled textures only allow [=read accesses=].
+
+The [=texel format=] of the texture is defined in the {{GPUTextureBindingLayout}}.
+
+<table class='data'>
+  <thead>
+    <tr><th>Type<th>[=texture/Dimensionality=]<th>[=texture/Arrayed=]
+  </thead>
+  <tr><td><dfn noexport dfn-for='type'>texture_1d</dfn><*T*>
+      <td>1D
```

Comment:
We should probably use the names from GPUTextureViewDimension, and link to individual names.

https://gpuweb.github.io/gpuweb/#enumdef-gputextureviewdimension 

and https://gpuweb.github.io/gpuweb/#dom-gputextureviewdimension-1d 


The WGSL definition of dimensionality should point to GPUTextureViewDimension.